src/main/java/enmasse/kafka/bridge/tracker/SimpleOffsetTracker.java

```diff
@@ -32,52 +32,53 @@
  */
 public class SimpleOffsetTracker<K, V> implements OffsetTracker<K, V> {
 
+	private static class PartitionState {
+		public long offset;
+		public boolean changed;
```

Comment:
I would usually agree, and I don't exactly disagree in this particular case. At the time of writing it was easier to go though statement by statement replacing the old code with the new using fields.

In this particular case the inner class never leaks outside the enclosing class, and the code is all pretty trivial: It's easy to see the field readers and writers because they're all in the same file. Adding getters and setter would just make the code longer for no benefit, imo. In a large class, or if the inner class was externally accessible I would certainly use accessors. 

I'll happily encapsulate if that's what you prefer.